// Validating benchmarks:
// ***** BenchmarkRunner: Start   *****
// ***** Found 9 benchmark(s) in total *****
// ***** Building 1 exe(s) in Parallel: Start   *****
// start dotnet restore  /p:UseSharedCompilation=false /p:BuildInParallel=false /m:1 /p:Deterministic=true /p:Optimize=true in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5
// command took 2,95s and exited with 0
// start dotnet build -c Release  --no-restore /p:UseSharedCompilation=false /p:BuildInParallel=false /m:1 /p:Deterministic=true /p:Optimize=true in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5
// command took 5,11s and exited with 0
// ***** Done, took 00:00:08 (8.21 sec)   *****
// Found 9 benchmarks:
//   AllBenchmarks.DictionaryTestSearch10000: DefaultJob
//   AllBenchmarks.DictionaryTestSearch50000: DefaultJob
//   AllBenchmarks.DictionaryTestSearch100000: DefaultJob
//   AllBenchmarks.BTreeTestSearch10000: DefaultJob
//   AllBenchmarks.BTreeTestSearch50000: DefaultJob
//   AllBenchmarks.BTreeTestSearch100000: DefaultJob
//   AllBenchmarks.BinaryTreeTestSearch10000: DefaultJob
//   AllBenchmarks.BinaryTreeTestSearch50000: DefaultJob
//   AllBenchmarks.BinaryTreeTestSearch100000: DefaultJob

// **************************
// Benchmark: AllBenchmarks.DictionaryTestSearch10000: DefaultJob
// *** Execute ***
// Launch: 1 / 1
// Execute: dotnet "a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.dll" --benchmarkName "Benchmark.AllBenchmarks.DictionaryTestSearch10000" --job "Default" --benchmarkId 0 in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\bin\Release\net6.0
// BeforeAnythingElse

System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at Main.DataBase.DictionaryLoading(Int32[] keys, Int32 n) in D:\C#\ProjectMyDictionary\Dictionary\DataBase.cs:line 32
   at Benchmark.AllBenchmarks..ctor() in D:\C#\ProjectMyDictionary\Benchmark\AllBenchmarks.cs:line 25
   at BenchmarkDotNet.Autogenerated.Runnable_0..ctor() in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.notcs:line 189
   at BenchmarkDotNet.Autogenerated.Runnable_0.Run(IHost host, String benchmarkName) in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.notcs:line 137
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Span`1& arguments, Signature sig, Boolean constructor, Boolean wrapExceptions)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at System.Reflection.MethodBase.Invoke(Object obj, Object[] parameters)
   at BenchmarkDotNet.Autogenerated.UniqueProgramName.AfterAssemblyLoadingAttached(String[] args) in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.notcs:line 59
// AfterAll
// Benchmark Process 9868 has exited with code -1.
No more Benchmark runs will be launched as NO measurements were obtained from the previous run!


// **************************
// Benchmark: AllBenchmarks.DictionaryTestSearch50000: DefaultJob
// *** Execute ***
// Launch: 1 / 1
// Execute: dotnet "a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.dll" --benchmarkName "Benchmark.AllBenchmarks.DictionaryTestSearch50000" --job "Default" --benchmarkId 1 in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\bin\Release\net6.0
// BeforeAnythingElse

System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at Main.DataBase.DictionaryLoading(Int32[] keys, Int32 n) in D:\C#\ProjectMyDictionary\Dictionary\DataBase.cs:line 32
   at Benchmark.AllBenchmarks..ctor() in D:\C#\ProjectMyDictionary\Benchmark\AllBenchmarks.cs:line 25
   at BenchmarkDotNet.Autogenerated.Runnable_1..ctor() in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.notcs:line 921
   at BenchmarkDotNet.Autogenerated.Runnable_1.Run(IHost host, String benchmarkName) in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.notcs:line 869
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Span`1& arguments, Signature sig, Boolean constructor, Boolean wrapExceptions)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at System.Reflection.MethodBase.Invoke(Object obj, Object[] parameters)
   at BenchmarkDotNet.Autogenerated.UniqueProgramName.AfterAssemblyLoadingAttached(String[] args) in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.notcs:line 59
// AfterAll
// Benchmark Process 8260 has exited with code -1.
No more Benchmark runs will be launched as NO measurements were obtained from the previous run!


// **************************
// Benchmark: AllBenchmarks.DictionaryTestSearch100000: DefaultJob
// *** Execute ***
// Launch: 1 / 1
// Execute: dotnet "a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.dll" --benchmarkName "Benchmark.AllBenchmarks.DictionaryTestSearch100000" --job "Default" --benchmarkId 2 in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\bin\Release\net6.0
// BeforeAnythingElse

System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at Main.DataBase.DictionaryLoading(Int32[] keys, Int32 n) in D:\C#\ProjectMyDictionary\Dictionary\DataBase.cs:line 32
   at Benchmark.AllBenchmarks..ctor() in D:\C#\ProjectMyDictionary\Benchmark\AllBenchmarks.cs:line 25
   at BenchmarkDotNet.Autogenerated.Runnable_2..ctor() in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.notcs:line 1653
   at BenchmarkDotNet.Autogenerated.Runnable_2.Run(IHost host, String benchmarkName) in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.notcs:line 1601
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Span`1& arguments, Signature sig, Boolean constructor, Boolean wrapExceptions)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at System.Reflection.MethodBase.Invoke(Object obj, Object[] parameters)
   at BenchmarkDotNet.Autogenerated.UniqueProgramName.AfterAssemblyLoadingAttached(String[] args) in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.notcs:line 59
// AfterAll
// Benchmark Process 6380 has exited with code -1.
No more Benchmark runs will be launched as NO measurements were obtained from the previous run!


// **************************
// Benchmark: AllBenchmarks.BTreeTestSearch10000: DefaultJob
// *** Execute ***
// Launch: 1 / 1
// Execute: dotnet "a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.dll" --benchmarkName "Benchmark.AllBenchmarks.BTreeTestSearch10000" --job "Default" --benchmarkId 3 in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\bin\Release\net6.0
// BeforeAnythingElse

System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at Main.DataBase.DictionaryLoading(Int32[] keys, Int32 n) in D:\C#\ProjectMyDictionary\Dictionary\DataBase.cs:line 32
   at Benchmark.AllBenchmarks..ctor() in D:\C#\ProjectMyDictionary\Benchmark\AllBenchmarks.cs:line 25
   at BenchmarkDotNet.Autogenerated.Runnable_3..ctor() in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.notcs:line 2385
   at BenchmarkDotNet.Autogenerated.Runnable_3.Run(IHost host, String benchmarkName) in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.notcs:line 2333
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Span`1& arguments, Signature sig, Boolean constructor, Boolean wrapExceptions)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at System.Reflection.MethodBase.Invoke(Object obj, Object[] parameters)
   at BenchmarkDotNet.Autogenerated.UniqueProgramName.AfterAssemblyLoadingAttached(String[] args) in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.notcs:line 59
// AfterAll
// Benchmark Process 13108 has exited with code -1.
No more Benchmark runs will be launched as NO measurements were obtained from the previous run!


// **************************
// Benchmark: AllBenchmarks.BTreeTestSearch50000: DefaultJob
// *** Execute ***
// Launch: 1 / 1
// Execute: dotnet "a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.dll" --benchmarkName "Benchmark.AllBenchmarks.BTreeTestSearch50000" --job "Default" --benchmarkId 4 in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\bin\Release\net6.0
// BeforeAnythingElse

System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at Main.DataBase.DictionaryLoading(Int32[] keys, Int32 n) in D:\C#\ProjectMyDictionary\Dictionary\DataBase.cs:line 32
   at Benchmark.AllBenchmarks..ctor() in D:\C#\ProjectMyDictionary\Benchmark\AllBenchmarks.cs:line 25
   at BenchmarkDotNet.Autogenerated.Runnable_4..ctor() in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.notcs:line 3117
   at BenchmarkDotNet.Autogenerated.Runnable_4.Run(IHost host, String benchmarkName) in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.notcs:line 3065
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Span`1& arguments, Signature sig, Boolean constructor, Boolean wrapExceptions)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at System.Reflection.MethodBase.Invoke(Object obj, Object[] parameters)
   at BenchmarkDotNet.Autogenerated.UniqueProgramName.AfterAssemblyLoadingAttached(String[] args) in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.notcs:line 59
// AfterAll
// Benchmark Process 12500 has exited with code -1.
No more Benchmark runs will be launched as NO measurements were obtained from the previous run!


// **************************
// Benchmark: AllBenchmarks.BTreeTestSearch100000: DefaultJob
// *** Execute ***
// Launch: 1 / 1
// Execute: dotnet "a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.dll" --benchmarkName "Benchmark.AllBenchmarks.BTreeTestSearch100000" --job "Default" --benchmarkId 5 in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\bin\Release\net6.0
// BeforeAnythingElse

System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at Main.DataBase.DictionaryLoading(Int32[] keys, Int32 n) in D:\C#\ProjectMyDictionary\Dictionary\DataBase.cs:line 32
   at Benchmark.AllBenchmarks..ctor() in D:\C#\ProjectMyDictionary\Benchmark\AllBenchmarks.cs:line 25
   at BenchmarkDotNet.Autogenerated.Runnable_5..ctor() in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.notcs:line 3849
   at BenchmarkDotNet.Autogenerated.Runnable_5.Run(IHost host, String benchmarkName) in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.notcs:line 3797
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Span`1& arguments, Signature sig, Boolean constructor, Boolean wrapExceptions)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at System.Reflection.MethodBase.Invoke(Object obj, Object[] parameters)
   at BenchmarkDotNet.Autogenerated.UniqueProgramName.AfterAssemblyLoadingAttached(String[] args) in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.notcs:line 59
// AfterAll
// Benchmark Process 8600 has exited with code -1.
No more Benchmark runs will be launched as NO measurements were obtained from the previous run!


// **************************
// Benchmark: AllBenchmarks.BinaryTreeTestSearch10000: DefaultJob
// *** Execute ***
// Launch: 1 / 1
// Execute: dotnet "a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.dll" --benchmarkName "Benchmark.AllBenchmarks.BinaryTreeTestSearch10000" --job "Default" --benchmarkId 6 in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\bin\Release\net6.0
// BeforeAnythingElse

System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at Main.DataBase.DictionaryLoading(Int32[] keys, Int32 n) in D:\C#\ProjectMyDictionary\Dictionary\DataBase.cs:line 32
   at Benchmark.AllBenchmarks..ctor() in D:\C#\ProjectMyDictionary\Benchmark\AllBenchmarks.cs:line 25
   at BenchmarkDotNet.Autogenerated.Runnable_6..ctor() in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.notcs:line 4581
   at BenchmarkDotNet.Autogenerated.Runnable_6.Run(IHost host, String benchmarkName) in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.notcs:line 4529
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Span`1& arguments, Signature sig, Boolean constructor, Boolean wrapExceptions)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at System.Reflection.MethodBase.Invoke(Object obj, Object[] parameters)
   at BenchmarkDotNet.Autogenerated.UniqueProgramName.AfterAssemblyLoadingAttached(String[] args) in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.notcs:line 59
// AfterAll
// Benchmark Process 11572 has exited with code -1.
No more Benchmark runs will be launched as NO measurements were obtained from the previous run!


// **************************
// Benchmark: AllBenchmarks.BinaryTreeTestSearch50000: DefaultJob
// *** Execute ***
// Launch: 1 / 1
// Execute: dotnet "a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.dll" --benchmarkName "Benchmark.AllBenchmarks.BinaryTreeTestSearch50000" --job "Default" --benchmarkId 7 in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\bin\Release\net6.0
// BeforeAnythingElse

System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at Main.DataBase.DictionaryLoading(Int32[] keys, Int32 n) in D:\C#\ProjectMyDictionary\Dictionary\DataBase.cs:line 32
   at Benchmark.AllBenchmarks..ctor() in D:\C#\ProjectMyDictionary\Benchmark\AllBenchmarks.cs:line 25
   at BenchmarkDotNet.Autogenerated.Runnable_7..ctor() in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.notcs:line 5313
   at BenchmarkDotNet.Autogenerated.Runnable_7.Run(IHost host, String benchmarkName) in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.notcs:line 5261
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Span`1& arguments, Signature sig, Boolean constructor, Boolean wrapExceptions)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at System.Reflection.MethodBase.Invoke(Object obj, Object[] parameters)
   at BenchmarkDotNet.Autogenerated.UniqueProgramName.AfterAssemblyLoadingAttached(String[] args) in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.notcs:line 59
// AfterAll
// Benchmark Process 21720 has exited with code -1.
No more Benchmark runs will be launched as NO measurements were obtained from the previous run!


// **************************
// Benchmark: AllBenchmarks.BinaryTreeTestSearch100000: DefaultJob
// *** Execute ***
// Launch: 1 / 1
// Execute: dotnet "a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.dll" --benchmarkName "Benchmark.AllBenchmarks.BinaryTreeTestSearch100000" --job "Default" --benchmarkId 8 in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\bin\Release\net6.0
// BeforeAnythingElse

System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at Main.DataBase.DictionaryLoading(Int32[] keys, Int32 n) in D:\C#\ProjectMyDictionary\Dictionary\DataBase.cs:line 32
   at Benchmark.AllBenchmarks..ctor() in D:\C#\ProjectMyDictionary\Benchmark\AllBenchmarks.cs:line 25
   at BenchmarkDotNet.Autogenerated.Runnable_8..ctor() in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.notcs:line 6045
   at BenchmarkDotNet.Autogenerated.Runnable_8.Run(IHost host, String benchmarkName) in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.notcs:line 5993
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Span`1& arguments, Signature sig, Boolean constructor, Boolean wrapExceptions)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at System.Reflection.MethodBase.Invoke(Object obj, Object[] parameters)
   at BenchmarkDotNet.Autogenerated.UniqueProgramName.AfterAssemblyLoadingAttached(String[] args) in D:\C#\ProjectMyDictionary\Benchmark\bin\Release\net6.0\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5\a0a8dda2-2f05-4bf6-9d08-846e907ebbe5.notcs:line 59
// AfterAll
// Benchmark Process 3208 has exited with code -1.
No more Benchmark runs will be launched as NO measurements were obtained from the previous run!


// ***** BenchmarkRunner: Finish  *****

// * Export *
  BenchmarkDotNet.Artifacts\results\Benchmark.AllBenchmarks-report.csv
  BenchmarkDotNet.Artifacts\results\Benchmark.AllBenchmarks-report-github.md
  BenchmarkDotNet.Artifacts\results\Benchmark.AllBenchmarks-report.html

// * Detailed results *
AllBenchmarks.DictionaryTestSearch10000: DefaultJob
Runtime = ; GC = 
There are not any results runs

AllBenchmarks.DictionaryTestSearch50000: DefaultJob
Runtime = ; GC = 
There are not any results runs

AllBenchmarks.DictionaryTestSearch100000: DefaultJob
Runtime = ; GC = 
There are not any results runs

AllBenchmarks.BTreeTestSearch10000: DefaultJob
Runtime = ; GC = 
There are not any results runs

AllBenchmarks.BTreeTestSearch50000: DefaultJob
Runtime = ; GC = 
There are not any results runs

AllBenchmarks.BTreeTestSearch100000: DefaultJob
Runtime = ; GC = 
There are not any results runs

AllBenchmarks.BinaryTreeTestSearch10000: DefaultJob
Runtime = ; GC = 
There are not any results runs

AllBenchmarks.BinaryTreeTestSearch50000: DefaultJob
Runtime = ; GC = 
There are not any results runs

AllBenchmarks.BinaryTreeTestSearch100000: DefaultJob
Runtime = ; GC = 
There are not any results runs

// * Summary *

BenchmarkDotNet=v0.13.1, OS=Windows 10.0.19043.1889 (21H1/May2021Update)
AMD Ryzen 7 3700U with Radeon Vega Mobile Gfx, 1 CPU, 8 logical and 4 physical cores
.NET SDK=6.0.400
  [Host] : .NET 6.0.8 (6.0.822.36306), X64 RyuJIT


|                     Method | Mean | Error | MValue | Rank |
|--------------------------- |-----:|------:|-------:|-----:|
|  DictionaryTestSearch10000 |   NA |    NA |     NA |    ? |
|  DictionaryTestSearch50000 |   NA |    NA |     NA |    ? |
| DictionaryTestSearch100000 |   NA |    NA |     NA |    ? |
|       BTreeTestSearch10000 |   NA |    NA |     NA |    ? |
|       BTreeTestSearch50000 |   NA |    NA |     NA |    ? |
|      BTreeTestSearch100000 |   NA |    NA |     NA |    ? |
|  BinaryTreeTestSearch10000 |   NA |    NA |     NA |    ? |
|  BinaryTreeTestSearch50000 |   NA |    NA |     NA |    ? |
| BinaryTreeTestSearch100000 |   NA |    NA |     NA |    ? |

Benchmarks with issues:
  AllBenchmarks.DictionaryTestSearch10000: DefaultJob
  AllBenchmarks.DictionaryTestSearch50000: DefaultJob
  AllBenchmarks.DictionaryTestSearch100000: DefaultJob
  AllBenchmarks.BTreeTestSearch10000: DefaultJob
  AllBenchmarks.BTreeTestSearch50000: DefaultJob
  AllBenchmarks.BTreeTestSearch100000: DefaultJob
  AllBenchmarks.BinaryTreeTestSearch10000: DefaultJob
  AllBenchmarks.BinaryTreeTestSearch50000: DefaultJob
  AllBenchmarks.BinaryTreeTestSearch100000: DefaultJob

// * Warnings *
Environment
  Summary -> Detected error exit code from one of the benchmarks. It might be caused by following antivirus software:
        - Windows Defender (windowsdefender://)
Use InProcessEmitToolchain or InProcessNoEmitToolchain to avoid new process creation.


// * Legends *
  Mean   : Arithmetic mean of all measurements
  Error  : Half of 99.9% confidence interval
  MValue : Modal value, see http://www.brendangregg.com/FrequencyTrails/modes.html
  Rank   : Relative position of current benchmark mean among all benchmarks (Arabic style)
  1 ns   : 1 Nanosecond (0.000000001 sec)

// ***** BenchmarkRunner: End *****
// ** Remained 0 benchmark(s) to run **
Run time: 00:00:03 (3.59 sec), executed benchmarks: 9

Global total time: 00:00:11 (11.81 sec), executed benchmarks: 9
// * Artifacts cleanup *
